lib/rules/route-path-style.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+const utils = require('../utils/utils');
+const ember = require('../utils/ember');
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+const ERROR_MESSAGE = 'Use kebab-case in route path.';
+
+module.exports = {
+  meta: {
+    docs: {
+      description: 'Enforces usage of kebab-case (instead of snake_case or camelCase) in route paths',
+      category: 'Best Practices',
+      recommended: false
+    },
+    fixable: null,
```

Comment:
this should be reasonably easy to fix, right? but I guess this is what you meant with "changing the behavior"?

Reply:
Converting a string to kebab case should be trivial, but yeah, I'd be worried that an autofixer for this rule would go against the following [eslint](https://eslint.org/docs/developer-guide/working-with-rules) principle, as it could break existing links/URLs:
> Avoid any fixes that could change the runtime behavior of code and cause it to stop working.